modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/PlaybackSettingsFragment.kt

```diff
@@ -155,7 +155,10 @@ class PlaybackSettingsFragment : BaseFragment() {
                         saved = settings.skipForwardInSecsObservable
                             .subscribeAsState(settings.getSkipForwardInSecs())
                             .value,
-                        onSave = settings::setSkipForwardInSec
+                        onSave = {
+                            settings.setSkipForwardNeedsSync(true)
+                            settings::setSkipForwardInSec.invoke(it)
```

Comment:
Why are we manually calling invoke instead of calling `settings.setSkipForwardInSec(it)`? Same question for the skip forward changes? What you have works fine, but it's harder to read IMO, so I was just curious if maybe there's a reason to prefer this that I wasn't thinking of.

Reply:
I don't prefer it either. Went ahead and switched to `settings.setSkipForwardInSec(it)`.